src/plugins/data/common/index_patterns/types.ts

```diff
@@ -186,10 +215,22 @@ export interface FieldSpec {
 
 export type IndexPatternFieldMap = Record<string, FieldSpec>;
 
+/**
+ * Static index pattern format
```

Comment:
If this is public docs, the term "Static index pattern" needs some clarfication IMO.
If @internal, all good :+1: